app/services/retention_schedules/anonymise_case_service.rb

```diff
@@ -0,0 +1,136 @@
+module RetentionSchedules
+  class CaseNotClosedError < StandardError
+    def message
+      "Case cannoted be destroyed as it is still open"
+    end
+  end
+
+  class NoRetentionScheduleError < StandardError
+    def message
+      "Cases without Retention Schedules cannot be destroyed"
+    end
+  end
+
+  class WrongCaseTypeError< StandardError
+    def message
+      "Non Offender SAR / Offender SAR Compaint cases cannot be destroyed"
+    end
+  end
+
+  class UnactionableStateError < StandardError
+    def message
+      "Case must have a Retention Schedule state of to_be_anonymised to be destroyed"
+    end
+  end
+
+  class AnonymiseCaseService
+
+
+    ANON_VALUES = {
+      case_reference_number: 'XXXX XXXX',
+      date_of_birth: Date.new(01, 01, 0001),
+      email: 'anon@email.com',
+      message: 'XXXX XXXX',
+      name: 'XXXX XXXX',
+      postal_address: 'XXXX XXXX',
+      previous_case_numbers: 'XXXX XXXX',
+      prison_number: 'XXXX XXXX',
+      requester_reference: 'XXXX XXXX',
+      subject: 'XXXX XXXX',
+      subject_address: 'XXXX XXXX',
+      subject_aliases: 'XXXX XXXX',
+      subject_full_name: 'XXXX XXXX',
+      third_party_company_name: 'XXXX XXXX',
+      third_party_name: 'XXXX XXXX',
+
+      # Offender SAR Complaint specific fields
+      gld_contact_email: 'XXXX XXXX',
```

Comment:
Does it need to comply to an email format, like the other attribute above `email: 'anon@email.com'`?